nestor_api/lib/git.py

```diff
@@ -7,6 +7,11 @@
 from nestor_api.utils.logger import Logger
 
 
+def clone(repository: str, remote_repository_url: str, branch_name: str = "master"):
+    """Clone a repository branch locally"""
+    return io.execute(f"git clone {remote_repository_url} -b {branch_name}", repository)
```

Comment:
`update_repository` already performs the `clone` operation. Not sure if we want to differentiate it or not.